.github/workflows/ci.yml

```diff
@@ -56,6 +56,8 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest]
+        toolchain:
+          - "1.62"
```

Comment:
That shouldn't be needed, since you aren't using `${{ matrix.toolchain }}` anywhere.